change/@microsoft-teams-js-828b759f-2466-4a17-9110-350aee8e7a35.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "change the constructor function to make the onRecordingStarted callback mandatory, make onRecordingStopped an optional property that can be pssed to the constructor, add appropriate tests",
```

Comment:
Maybe don't need "added tests" in the patch notes

Reply:
cool, will update!  thans!